### PR TITLE
Svnsairam setup turtlestack cli

### PR DESCRIPTION
**Pull request by svnsairam**

### CODE REVIEW

**Comment by TheIronDev** [comment]
I believe you're going to want to feature-gate this.

File: src/commands/init.ts [hunk 1]
```diff
@@ -70,6 +70,11 @@ const choices = [
     name: "Extensions: Set up an empty Extensions manifest",
     checked: false,
   },
+  {
+    value: "frameworkstack",
```
Comment (taeold) [inline src/commands/init.ts:74]: can I suggest the name "frameworks" instead?

File: src/commands/init.ts [hunk 2]
```diff
@@ -70,6 +70,11 @@ const choices = [
     name: "Extensions: Set up an empty Extensions manifest",
     checked: false,
   },
+  {
```
Comment (taeold) [inline src/commands/init.ts:73]: this should be behind a experiment flag - we don't want users to see this as a choice.

Checkout https://github.com/firebase/firebase-tools/pull/5465 to see an example of how you can create a new experimental feature and to use it to change behavior of init!

File: src/init/features/frameworkstack/index.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      type: "input",
+      default: "acme-inc-web",
+      message: "Create a name for your service [1-64 characters]",
+    },
+    setup.frameworkstack
+  );
+
+  await promptOnce(
+    {
+      name: "regionName",
```
Comment (taeold) [inline src/init/features/frameworkstack/index.ts:41]: nit* prefer `region` over `regionName`

**Pull request closed without merging**